lib/ansible/modules/fail.py

```diff
@@ -38,8 +38,8 @@
 '''
 
 EXAMPLES = r'''
-# Example playbook using fail and when together
-- fail:
+- name: Example playbook using fail and when together
```

Comment:
This is a task, not a playbook :)
```suggestion
- name: Example using fail and when together
```